deblur/workflow.py

```diff
@@ -576,6 +577,10 @@ def get_files_for_table(input_dir,
     file_end : string
         the ending of all the fasta files to be added to the table
         (default '.fasta.trim.derep.no_artifacts.msa.deblur.no_chimeras')
+    num_extra_name_chars : int (optional)
+        the number of additional characters to remove from the end of the
+        file name to get the sample name
+        default is len('.fasta')=6
```

Comment:
I think asking the user to count characters is not the best way to move forward. If we know that at this point the files are not compressed, we can safely used `os.path.splitext` which will remove the extension. (it will be similar to a rsplit, but safer, you can check the documentation: https://docs.python.org/2/library/os.path.html#os.path.splitext)